index.html

```diff
@@ -13,20 +13,53 @@
     .attribution { font-size: 11px; text-align: center; }
     .attribution a { color: hsl(228, 45%, 44%); }
   </style>
+  <link rel="stylesheet" href="./styles.css">
+  <!-- font 1 -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lexend+Deca&display=swap" rel="stylesheet">
+  <!-- font 2 -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Big+Shoulders+Display:wght@700&display=swap" rel="stylesheet">
 </head>
 <body>
-  
-  Sedans
-  Choose a sedan for its affordability and excellent fuel economy. Ideal for cruising in the city 
-  or on your next road trip.
+  <main>
+    <div class="columns">
+  <div class="column-container column-sedans">
+    <picture class="icon">
+      <img src="./icon-sedans.svg" alt="sedan icon">
+    </picture>
+    <h1 class="car-type">Sedans</h1>
+      <p class="car-description">Choose a sedan for its affordability and excellent fuel economy. Ideal for cruising in the city 
+    or on your next road trip.
+      </p>
+      <button class="learn-sedans" src="#">Learn more</button>
```

Comment:
I can see that you pass `src` attribute. If the intention is that learn more button would navigate you to another page, then the correct semantic HTML element to use is actually an anchor tag `a`. Buttons are better suited for actions on the same page (e.g. submitting a form, copying a text to clipboard, opening/closing a dialog).

Note on the accessibility of `a` elements containing "Learn more". It's a good idea to add more context to each link for assistive technologies (AT) such as screen readers because they have an outline of all links on the page. In this outline links are out of context, thus, the user gets three learn more links and can't distinguish between them.

We can add more context using the [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) attribute. So a screen reader will read out and add the text inside the aria-label to the outline, whereas users without a screen reader will still see "Learn more" text.


```suggestion
      <a class="learn-sedans" src="#" aria-label="Learn more about Sedans">Learn more</a>
```

Note: you don't need to add aria-label to all links. For example the Frontend Mentor link doesn't need it because it is distinguishable by itself.

Reply:
We tried to change it from <button> to <a>, but then the button on the picture would disappear.

Reply:
Did you try updating the style of the `<a>` element so it looks like a button?